central/metrics/aggregator/common/common.go

```diff
@@ -0,0 +1,64 @@
+package common
+
+import (
+	"context"
+	"iter"
+	"regexp"
+	"slices"
+
+	"github.com/pkg/errors"
+	v1 "github.com/stackrox/rox/generated/api/v1"
+	"github.com/stackrox/rox/pkg/logging"
+)
+
+var (
+	log = logging.CreateLogger(logging.ModuleForName("central_metrics"), 1)
+
+	metricNamePattern = regexp.MustCompile("^[a-zA-Z0-9_]+$")
```

Comment:
could you link to https://prometheus.io/docs/concepts/data_model/#metric-names-and-labels:~:text=names%20SHOULD%20match%20the-,regex,-%5Ba%2DzA%2DZ_%3A%5D%5Ba
```suggestion
	metricNamePattern = regexp.MustCompile("[a-zA-Z_:][a-zA-Z0-9_:]*")
```